app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<p> list of all posts and comment of a user </p>
\ No newline at end of file
+<h2> list of all posts and comment of a user </h2>
+
+ <div class = 'posts-content'>
+     <% @posts.each do |post|%>
+     <div class = 'user-post'>
+     <div class = 'post-content'> 
+     <li><h4>   <%= link_to post.title, user_post_path(user_id: @user.id, id: post.id) %>
+    </h4></li>
+     <p> <%= post.text%> </p>
+     <p class = 'counters'> Comments: <%= post.comments_counter.to_i > 0 ? post.comments_counter : '0'  %> Likes : <%= post.likes_counters.to_i > 0 ? post.likes_counters : '0'%>  </p>
+     </div>
+     <div class='comments-content'>
+     <p><strong> Comments :</strong> </p>
+     <% post.comments.each do |comment|%>
```

Comment:
- [ ] Once more, we need only the first five comments on this page not all comments. Remember you can utilize your awesome `#recent_comments` method in your Post model for this purpose :+1: Only the post show page should display all comments for a post :+1: